Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -425,6 +425,34 @@ public extension String {
 		// https://stackoverflow.com/questions/42822838
 		return words().count
 	}
+
+    /// SwifterSwift: Transforms the string into a slug string.
+    ///
+    ///        "Swift is amazing".toSlug() -> "swift-is-amazing"
+    ///
+    /// - Returns: The string in slug format.
+    public func toSlug() -> String {
```

Comment:
I've never heard of a "slug" before 😂 But if this is something people need then it looks 👌

Reply:
Slugs are used (in my experience) as safe endpoints to urls for blog posts etc.. Wordpress is all about the slug.

Reply:
Ahhh, I see. Thanks for the info @wdcurry 👍